documentation/modules/rhv-prerequisites.adoc

```diff
@@ -8,6 +8,9 @@
 
 The following prerequisites apply to {rhv-full} migrations:
 
+* You must have one of the following:
+** {rhv-short} admin permissions. These permissions allow you to migrate any virtual machine in the system.
+** `DiskCreator` and `UserVmManager` permissions: These permissions have to be granted to you individually by an `admin` user per each individual virtual machine that you want to migrate.
```

Comment:
Suggestions:
1. End "DiskCreator` and `UserVmManager` permissions:" with a period as you did with RHV admin permissions. Alternatively, use a colon after "permissions" in both cases, although I think a period is better. 
2. I don't think you need the word "individually" in "granted to you individually, " unless you want to say it can't be granted by a role-based access control (RBAC) permission set.
3. You probably don't need "individual"  in "each individual VM" either, although in the latter case, it does emphasize that permission is granted per machine, and nit for a group of them, so I'd say there's a good case for leaving it.

Reply:
@RichardHoch 
1. Fixed, thank you
2, 3. Yes, that's the point of the option - the permissions should be granted to an individual user, not to a role, by an admin and specifically for each virtual machine. So I think I have to keep 'individually'. Are you OK with that?

Reply:
OK.

Reply:
LGTM